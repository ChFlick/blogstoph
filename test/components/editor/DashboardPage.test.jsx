import React from 'react';
import { shallow } from 'enzyme';
import EditorDashboardPage from '../../../src/components/editor/DashboardPage';

test('should render the dashboard page correctly', () => {
    const wrapper = shallow(<EditorDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});
