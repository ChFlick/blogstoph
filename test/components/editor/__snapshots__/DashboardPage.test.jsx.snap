// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the dashboard page correctly 1`] = `
<div>
  <h1>
    Editor Dashboard
  </h1>
  <Connect(PostList) />
  <Link
    replace={false}
    to="./addPost"
  >
    Add Post
  </Link>
</div>
`;
